redash/models.py

```diff
@@ -748,7 +750,7 @@ def should_schedule_next(previous_iteration, now, schedule, failures):
 
 class Query(ChangeTrackingMixin, TimestampMixin, BelongsToOrgMixin, db.Model):
     id = Column(db.Integer, primary_key=True)
-    version = Column(db.Integer, default=1)
+    version = Column(db.Integer, default=0)
```

Comment:
Are there any current instances that have this set to 1 that would get an upgrade bug/unexpected behavior because of this?